Makefile

```diff
@@ -147,12 +147,28 @@ java-image:
 # Push the Java runtime image to a docker registry
 push-java-image:
 	docker push ${IMAGE_PREFIX}java:${TEST_INFRA_VERSION}
-
 # Build the Ruby build image
 ruby-build-image:
 	docker build -t ${BUILD_IMAGE_PREFIX}ruby:${TEST_INFRA_VERSION} \
 		containers/init/build/ruby
 
+# Push the PHP runtime image to a docker registry
+push-php-build-image:
+	docker push ${BUILD_IMAGE_PREFIX}php:${TEST_INFRA_VERSION}
+
+# Build the PHP runtime image
+php-image:
+	docker build -t ${IMAGE_PREFIX}php:${TEST_INFRA_VERSION} \
+		containers/runtime/php
+
+# Push the PHP runtime image to a docker registry
+push-php-image:
+	docker push ${IMAGE_PREFIX}php:${TEST_INFRA_VERSION}
+# Build the PHP build image
```

Comment:
nit: missing a space between lines 166 and 167. In addition, let's keep push- and the build of the images together. So, the target push-ruby-build-image will follow ruby-build-image and push-php-image will follow php-image and so on.

Reply:
Right, thanks for pointing this out!

Reply:
nit: I'm not sure if this is resolved. I think images and their push should be adjacent. So, for example, the `push-php-build-image` target is defined in line 156. The image that it is pushing, `php-build-image` is defined in line 169. I'd like to keep them together, in an order, like-so:

```
php-build-image:
push-php-build-image:
php-image:
push-php-image:
```

This ordering makes the file a little easier to read and comprehend.